app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,7 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if can? :access, Admin
```

Comment:
It seems to me that this file would be better structured if the links in it appeared (or not) independently of any logic there is behind our application.
That way even if we change some logic in the application, we don't have to worry about touching the code for this menu.

This if statement *implies* that if you have access to the admin area you can properly access the links inside the if statement (links redirect to **show** action for conference and **new** action for conference), which is of course not true.

Even though our current application logic might implicitely prevent us from showing the **new** link when it wouldn't be accessible, in my opinion, this is more of a happy coincidence rather than an explicit statement with code.